src/main/resources/db/migration/V1_25__add_question_label_to_questions.sql

```diff
@@ -0,0 +1,8 @@
+ALTER TABLE question ADD COLUMN label TEXT;
+ALTER TABLE historical_question ADD COLUMN label TEXT;
+
+update question set label = question;
+update historical_question set label = question;
+
+ALTER TABLE question ALTER COLUMN label SET NOT NULL;
+ALTER TABLE historical_question ALTER COLUMN label SET NOT NULL;
```

Comment:
Instead of altering the column created in this PR, I'd just change the `ADD COLUMN` statements above to add `NOT NULL` columns instead.

(Same result but tidier migration)